tests/Microsoft.Bot.Builder.Prompts.Tests/ConfirmPromptTests.cs

```diff
@@ -95,5 +95,119 @@ public class ConfirmPromptTests
                 .StartTest();
         }
 
+
+        [TestMethod]
```

Comment:
Nice set of tests!

Reply:
With all the changes, I needed to fix existing tests, so assert of messages will have sense again, and pass.